keyboards/s7_elephant/rev2/info.json

```
@@ -0,0 +1,90 @@
+{
+    "keyboard_name": "S7 Elephant",
+    "url": "https://geekhack.org/index.php?topic=103148.0",
+    "maintainer": "qmk",
+    "width": 17.25,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
```

you're missing the LAYOUT_all and LAYOUT_ansi which you defined in rev2.h

---

What's the best way to generate an info.json? Adding those two layouts by hand doesn't sound like the correct methodology

---

Noroadsleft tool only does it for the layout you created on kbfirmware. You can go about creating each layout again and just copy pasting the resulting LAYOUT macro, or you can start with the base .h file and modify it from there

---

This actually matches the `LAYOUT_all` macro.

```suggestion
        "LAYOUT_all": {
```

---

Yeah it does, I need to add the LAYOUT_ansi as well though, I'll get to that tonight